### DISCUSSION TITLE
Couple issues with secondary monitor and start button and autohiding taskbar screws up alignment and doesn't allow start menu to display

### DISCUSSION THREAD

**Discussion by fihnakis** [valinet/ExplorerPatcher#2506] [General]
Per title, issues in order with examples.

When using secondary monitor to display start menu and you use the windows key to display start menu, it opens halfway off the screen.
![start button](https://github.com/valinet/ExplorerPatcher/assets/30006560/6c5ad3d3-e34a-49da-a058-e8b5a07b188e)

When autohiding the taskbar, the taskbar is pushed out of alignment with start button and using the windows key does not display the start menu. If you unlock the taskbar, the alignment is corrected but start menu still does not display when using windows key. If you drag mouse down to unhide the taskbar and press windows key, start menu displays but off screen as in my first posted issue.
![autohide](https://github.com/valinet/ExplorerPatcher/assets/30006560/e73ba5b0-b81d-4e87-b21f-ab2eba8fb41f)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Comment by fihnakis** [comment]
Fresh install of Win11 ver 23H2 Build 22631.2715 and EP 22621.2506.60.1

Only settings changed were taskbar style windows 10 so I can move it to secondary monitor, primary and secondary alignment centered with start button, always combine for both options, and monitor #2 for where to display start menu.

**Comment by pyrates999** [comment]
do you have the option **register as file extension** enabled?

**Reply by fihnakis** [reply]
I tried enabling that option and it didn't make a difference, still same issues.

**Reply by pyrates999** [reply]
Disable that option if you have it enabled.

**Reply by fihnakis** [reply]
I re-disabled it once I found it didn't change anything. Hopefully it's something that gets fixed as EP evolves.

**Comment by fihnakis** [comment]
I do not, should I?

**Comment by pyrates999** [comment]
Here's why you shouldn't have it enabled: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension
